Source/DataTypes/SvgUnit.cs

```diff
@@ -164,6 +164,14 @@ public float ToDeviceValue(ISvgRenderer renderer, UnitRenderingType renderType,
                         case UnitRenderingType.VerticalOffset:
                             _deviceValue = (size.Height / 100) * value + boundable.Location.Y;
                             break;
+                        case UnitRenderingType.Other:

```

Comment:
This actually replaces the default branch, so you can remove the default as far as I can see.